### PR TITLE
add duration in isochrone request

### PR DESCRIPTION
**Pull request by ZoeBrunet**
Add arguments for multi isochrones

### CODE REVIEW

File: request.proto [hunk 1]
```diff
@@ -166,6 +166,7 @@ message NearestStopPointsRequest {
 message GraphicalIsochroneRequest {
     optional JourneysRequest journeys_request = 1;
     required int32 min_duration               = 2;
+    repeated int32 duration                   = 4;
```
Comment (pbougue) [inline request.proto]: `duration` seems ambiguous to me, no?

Comment (TeXitoi) [inline request.proto]: That's the API parameter. If that's not clear, maybe we should modify the API parameter. @stifoon ?

File: request.proto [hunk 2]
```diff
@@ -166,6 +166,7 @@ message NearestStopPointsRequest {
 message GraphicalIsochroneRequest {
     optional JourneysRequest journeys_request = 1;
     required int32 min_duration               = 2;
```
Comment (TeXitoi) [inline request.proto]: min_duration should be removed, no?

**Comment by TeXitoi** [comment]
OK for me. @pbougue OK for you?

File: request.proto [hunk 3]
```diff
@@ -165,7 +165,7 @@ message NearestStopPointsRequest {
 
 message GraphicalIsochroneRequest {
     optional JourneysRequest journeys_request = 1;
-    required int32 min_duration               = 2;
+    repeated int32 duration                   = 4;
```
Comment (pbougue) [inline request.proto]: still really not fan of the name (but the problem is more on params) and it should be `durations` at least, no? `duration_bounds` would be nicer I think.

Comment (TeXitoi) [inline request.proto]: Every `param[]` is singular except `forbidden_uris[]`. That's why `duration[]` is singular.

Comment (pbougue) [inline request.proto]: @stifoon : any opinion?

Comment (stifoon) [inline request.proto]: I prefer step_duration, as :
https://api.navitia.io/.../isochrones?from=..&to=...&min_duration=0&step_duration[]=600&step_duration[]=1200&step_duration[]=2400&step_duration[]=3600&max_duration=7200

I think the URI is more blue like that.

**Pull request merged**